capabilities/go.mod

```diff
@@ -0,0 +1,78 @@
+module github.com/smartcontractkit/chainlink-framework/capabilities
+
+go 1.24.1
+
+require (
+	github.com/google/uuid v1.6.0
+	github.com/jpillora/backoff v1.0.0
+	github.com/smartcontractkit/chainlink-common v0.6.1-0.20250407100046-dfdf9600557b
+	github.com/smartcontractkit/chainlink-evm/capabilities v0.0.0-20250408001113-f835ed8de109
```

Comment:
If we import evm here, then we have the same problem as before. Perhaps I misunderstood something, but the goal we are trying to achieve with the separation is to keep this repo chain agnostic.

Reply:
Right now, the report decoding in the generalized write target is tightly coupled to the evm code, and it will take some effort to decouple those to completely extract this from this repo. I have a [ticket here](https://smartcontract-it.atlassian.net/browse/BCFR-1221) to fix that in the future, but it I am not confident I can get that done before the release today. Therefore, I think we should keep this the way it is for now and I will fix it after the release cut. Is that acceptable?

Reply:
I think this is the same issue as before, and a separate one from generalizing more things. Anything that is not ready to be chain agnostic as part of this current PR simply doesn't belong in this repo. We can always move things here later when they are made agnostic, but doing it prematurely pollutes the import graph (creating a repo cycle in this case) and affects all dependents.

Reply:
No that is different and I don't understand why that is an issue. You are creating new modules so nothing that currently exists becomes entangled, and there would be no repo cycles.

Reply:
Ok, I can move all the EVM related code to `chainlink-evm`. Although it is not a new module anymore after some other reviews, but I don't think that will be an issue.